provisioning/JJG-Ansible-Windows/windows.sh

```diff
@@ -17,7 +34,7 @@ YUM=$(which yum 2>/dev/null)
 APT_GET=$(which apt-get 2>/dev/null)
 
 # Make sure Ansible playbook exists.
-if [ ! -f "/vagrant/$ANSIBLE_PLAYBOOK" ]; then
```

Comment:
Ah yes we do. I'll submit a PR tonight. I have some other things to take care of right now but I'll have my own sprint in the evening.

Maybe we technically can avoid it but I feel it adds a lot more readability to what those variables are.